src/style.css

```
@@ -1 +1,61 @@
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding-left: 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding-top: 10px;
+}
+
+.nav__list {
+  box-sizing: border-box;
+  height: 60px;
+  display: flex;
+}
+
+.link {
+  line-height: 60px;
+  text-decoration: none;
+  color: #000;
+  font-weight: 500;
+  margin-right: 20px;
+}
+
+.link:last-child {
+  padding-right: 30px;
+}
```

```suggestion
.link:last-child {
  margin-right: 0;
}
```
Тут треба обнулити марджин для останнього елементу, а відступ у 50 пікселів задай падінгами для хедера. Зараз у тебе там тільки padding-left, але має бути аналогічний і справа